onnx/onnx-ml.proto

```diff
@@ -236,13 +236,10 @@ message GraphProto {
   // The name of the graph.
   optional string name = 2;   // namespace Graph
 
-  // A list of named tensor values (constants), used to specify default
+  // A list of named tensor values (constants), used to specify constant
   // values for some of the inputs of the graph.
   // Each TensorProto entry must have a distinct name (within the list) that
   // also appears in the input list.
-  // In an evaluation, the default value specified here is used if and only if
```

Comment:
I think this paragraph says that even the graph contains some values in the initializer, we can still ignore it by passing the value in the input list.

Reply:
@houseroad Yeah that was in the old days we were trying to provide the convenience to users to override the constant values in the `run_model`, `run_node` python interface, but it doesn't work anymore as we have more and more stuffs relying on the static graph analysis.